### PR TITLE
finish Python 2 -> 3 migration, remove dependecy on Six

### PR DESCRIPTION
**Pull request by a-detiste**

### CODE REVIEW

**Review by rafaelweingartner — APPROVED** [review]

File: gnocchiclient/tests/functional/base.py [hunk 1]
```diff
@@ -73,8 +69,6 @@ def _run(self, binary, action, flags='', params='',
                                cmd,
                                result,
                                result_err)
-        if not six.PY2:
```
Comment (a-detiste) [inline gnocchiclient/tests/functional/base.py:76]: Whoops

**Review by a-detiste — COMMENTED** [review]
****

**Comment by tobias-urdin** [comment]
@mergifyio rebase

**Review by chungg — APPROVED** [review]

**Review by tobias-urdin — APPROVED** [review]

**Comment by tobias-urdin** [comment]
thanks!

**Pull request merged**